cmd/packages.go

```diff
@@ -260,21 +261,29 @@ func packagesExecWorker(userInput string) <-chan error {
 		}
 
 		if appConfig.Anchore.Host != "" {
-			if err := runPackageSbomUpload(src, src.Metadata, catalog, d, appConfig.Package.Cataloger.ScopeOpt); err != nil {
+			if err := runPackageSbomUpload(src, src.Metadata, catalog, d); err != nil {
 				errs <- err
 				return
 			}
 		}
 
+		sbomResult := sbom.SBOM{
+			Artifacts: sbom.Artifacts{
+				PackageCatalog: catalog,
+				Distro:         d,
+			},
+			Source: src.Metadata,
+		}
+
 		bus.Publish(partybus.Event{
 			Type:  event.PresenterReady,
-			Value: f.Presenter(catalog, &src.Metadata, d, appConfig.Package.Cataloger.ScopeOpt),
+			Value: f.Presenter(sbomResult),
 		})
 	}()
 	return errs
 }
 
-func runPackageSbomUpload(src *source.Source, s source.Metadata, catalog *pkg.Catalog, d *distro.Distro, scope source.Scope) error {
+func runPackageSbomUpload(src *source.Source, s source.Metadata, catalog *pkg.Catalog, d *distro.Distro) error {
```

Comment:
**Question:** Would we expect to see this function's parameter list start using the unified SBOM type as well?

Reply:
Would we still need `*source.Source` passed in? so `(src *source.Source, sbom.SBOM)`

I don't see anywhere else in the code where those two types are paired up which gives me some pause.

If we assemble the SBOM type and then just have to deconstruct its parts for the ImportConfig then in my mind its a bit clearer to have them separated in the signature so there is one less indirection.

```
anchore.ImportConfig{
    Catalog: sbom.Artifacts.PackageCatalog
}
```

vs

```
	importCfg := anchore.ImportConfig{
		Catalog:                 catalog,
	}
```

Reply:
@wagoodman this was the only outstanding question left here

Reply:
I hesitated with updating the import path with the same pattern --but in hind sight I should have. I'll update in a follow up PR.

Reply:
Follow up PR https://github.com/anchore/syft/pull/621